scholarx/archive/functions.js

```diff
@@ -60,16 +60,22 @@ async function getData() {
 
 async function loadData() {
     const { data } = await getData();
+    const mentorNames = new Set(); // Set to store unique mentor names
     for (let i = 0; i < data.length; i++) {
-        years.push(data[i].year)
-        industries.push(data[i].fields)
+        years.push(data[i].year);
+        industries.push(data[i].fields);
         if (data[i].type == "mentor") {
-            mentors.push(data[i])
+            // Check if the mentor's name is already in the Set
+            if (!mentorNames.has(data[i].name)) {
+                mentorNames.add(data[i].name); // Add the mentor's name to the Set
+                mentors.push(data[i]); // Add the mentor to the mentors array
+            }
```

Comment:
Good stuff @RamithaHeshan33. Can we make the mentors array a set without the mentorNames?

Reply:
> Good stuff @RamithaHeshan33. Can we make the mentors array a set without the mentorNames?

Yes @anjula-sack, I also understood that after submitting the pull request. Sorry about that. We can do the same thing through the mentor's title and mentor's bio. May I use the mentor's title to do this?

Reply:
I think the name is best.

Reply:
![update](https://github.com/sef-global/sef-site/assets/142712365/d6f5a5eb-9bba-4bad-80b8-02b8027288be)

How about this?